src/ReadConfig.cxx

```diff
@@ -34,6 +34,14 @@
 
 #endif
 
+#ifdef _WIN32
+#define _S_IFMT 0xF000
+#define _S_IFREG 0x8000
```

Comment:
Where do these "magic" values come from? Do they work on Windows?
Maybe, instead of using `stat()` on Windows with these obscure definitions, we should reimplement `file_exists()` with Windows-specific functions?